tests/installation/partitioning_lvm.pm

```diff
@@ -75,6 +87,48 @@ sub run {
             $collect_logs = 1;
         }
     }
+    elsif (get_var("STORAGE_NG")) {
+        if ($numdisks <= 1) {
+            die "Guided workflow does not skip disk selection when only one disk!"
+                if !match_has_tag('inst-partitioning-scheme');
```

Comment:
It's all subjective, but I totally agree with subpoints: Use unless and until only with positive statements so you don't create a double negative logic, which would be twice as confusing! Never follow an unless block with an else block.
If it's used in that way that you can read it in plain English, like "die unless" it's more wordy than "die if not", which is totally perlish and is confusing for some people. I'm fine if we are just consistent, so I do not request a change here. And thanks for the link, it's nice article!